prototype/tests/conftest.py

```diff
@@ -6,19 +6,48 @@
 from mobile_endpoint import create_app
 from mobile_endpoint.models import db
 
+couch = pytest.mark.couch
+sql = pytest.mark.sql
+rowsize = pytest.mark.rowsize
+
+
+def pytest_addoption(parser):
+    parser.addoption("--rowsize", action="store", metavar="model",
+        help="only run row size tests")
+
+
+def pytest_runtest_setup(item):
+    rowsize_marker = item.get_marker("rowsize")
+    run_rowsize_tests = item.config.getoption("--rowsize")
+    if rowsize_marker:
+        model = rowsize_marker.args[0]
+        if not run_rowsize_tests:
+            pytest.skip("need --rowsize option to run")
+        if model != run_rowsize_tests:
+            pytest.skip("only running tests for model: {}".format(run_rowsize_tests))
+    elif run_rowsize_tests:
+        pytest.skip("only running rowsize tests")
+
 
 @pytest.fixture(scope="session")
 def testapp():
-    from mobile_endpoint.backends.couch.db import init_dbs
     app = create_app('testconfig.py')
 
     db.app = app
-    with app.app_context():
+    return app
+
+
+@pytest.fixture(scope="session")
+def sqldb(testapp):
+    with testapp.app_context():
         upgrade()
-        # init couch DBs
-        init_dbs()
 
-    return app
+
+@pytest.fixture(scope="session")
+def couchdb(testapp):
```

Comment:
I split this out so that we don't have to do the couch setup if only running sql tests